lib/data/db.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:sqflite/sqflite.dart';
+import 'package:whitmaps/models/contact.dart';
+import 'package:whitmaps/models/poi.dart';
+
+
+class DB {
+  static createDatabase() async {
+    String databasesPath = await getDatabasesPath();
+    String dbPath = databasesPath + 'my.db';
+
+    var database = await openDatabase(dbPath, version: 1, onCreate: initDB);
+    return database;
+    }
+
+  static void initDB(Database db, int version) async {
+    await db.execute("CREATE TABLE Customer ("
+          "id INTEGER PRIMARY KEY,"
+          "first_name TEXT,"
+          "last_name TEXT,"
+          "email TEXT"
+          ")");
+  }
+
+  //-----------------------------------------------------------
+  //                       CONTACT
+  //-----------------------------------------------------------
+
+  static Future<int> createContact(Database db, Contact contact) async {
+    var result = await db.insert("Contact", contact.toMap());
+    return result;
+  }
+
+  static Future<List> getContacts(Database db) async {
+    var result = await db.query("Contact", columns:
+    ["name", "primary_phone", 
+    "home_phone", "mobile_phone", "office_phone"]);
+    return result.toList();
+  }
+
+  static Future<Contact> getContact(Database db, String name) async {
+    List<Map> results = await db.query("Contact",
+      columns: ["name", "primary_phone", "home_phone", "mobile_phone", "office_phone"],
+      where: 'name = ?',
```

Comment:
if both name and primary_phone are the primary key, then this needs to take both as an argument, as well as updating the sql 'where' clause to match